tests/cli_test.py

```diff
@@ -13,7 +13,7 @@
 
 def _lines(tbl):
     """Convert table to list of lines."""
-    return map(str.strip, str(tbl).splitlines())
+    return list(map(str.strip, str(tbl).splitlines()))
```

Comment:
More often than not we don't have to typecast to list but in this case it would make sense to return a list for comparison to avoid doing it multiple times in the assertions.

We've removed `list(...)` in other places where it doesn't make sense.